fakebook/src/app/component/comment-view/comment-view.component.html

```diff
@@ -5,7 +5,7 @@
           <img class="rounded-circle img-sm" alt="image" src="{{user.profilePictureUrl}}">
         </a>
         <div class="media-body">
-          <a href="/profile/{{comment.userEmail}}">
+          <a routerLink='/profile/{{comment.userEmail}}'>
```

Comment:
are you sure this works? i don't know if interpolation can be used nested inside attribute values. maybe you can.
use double quotes, not single quotes. it's done like this: `[routerLink]="['/profile', comment.userEmail]"`

Reply:
I saw both implementations when I searched it, and I checked the one from my groups p2, but ill do it the way you suggested